### ISSUE TITLE
CLI error with postgres: InvalidData: data did not match any variant of untagged enum ArgsEnum

### ISSUE THREAD

**Issue by quaos** [denolib/typeorm#90]
**Issue type:**

[ ] question
[x] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[x] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[ ] `latest`
[ ] `@next`
[x] `0.2.23-rc5` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

* Install TypeORM CLI
* Create a project
* Prepare some PostgresQL test database
* Set up environment variables:
```shell
$ env |grep TYPEORM_
TYPEORM_HOST=localhost
TYPEORM_CONNECTION=postgres
TYPEORM_PORT=5432
TYPEORM_USERNAME=****
TYPEORM_PASSWORD=****
TYPEORM_DATABASE=testdb
TYPEORM_ENTITIES_DIR=src/entities
TYPEORM_MIGRATIONS_DIR=src/migrations
TYPEORM_MIGRATIONS_TABLE_NAME=__typeorm_migrations
TYPEORM_LOGGING=true
TYPEORM_SYNCHRONIZE=false
```
* Try running TypeORM CLI commands:
```shell
$ typeorm migration:run
Error during migration run:
InvalidData: data did not match any variant of untagged enum ArgsEnum
    at unwrapResponse ($deno$/ops/dispatch_json.ts:42:11)
    at Object.sendAsync ($deno$/ops/dispatch_json.ts:93:10)
    at async Object.connect ($deno$/net.ts:183:11)
    at async Connection.startup (connection.ts:138:17)
    at async Pool._createConnection (pool.ts:32:5)
    at async pool.ts:61:7
    at async Promise.all (index 0)
    at async Pool._startup (pool.ts:63:25)
    at async Pool.connect (pool.ts:85:5)
    at async PostgresDriver.createPool (PostgresDriver.ts:944:28)
$ typeorm migration:show
# Same error
$ typeorm cache:clear
# Same error
```
* Now, try changing `TYPEORM_CONNECTION` to `sqlite` and run `typeorm migration:run` again. There'd be no error, with a sqlite db file created.

<!--
    To answer those questions you need to put "x" inside the square brackets, for example:
    [x] `mysql`
    [ ] `postgres`

    Also, please format your code properly (by taking code blocks into ```ts .... ```)
!>

**Comment by uki00a** [comment]
Thanks for opening this issue!
I was able to reproduce the same issue.
I'll look into it.

**Comment by uki00a** [comment]
This problem has been fixed at [v0.2.23-rc6](https://github.com/denolib/typeorm/releases/tag/v0.2.23-rc6)!

**Comment by quaos** [comment]
> This problem has been fixed at [v0.2.23-rc6](https://github.com/denolib/typeorm/releases/tag/v0.2.23-rc6)!

Thanks!
